src/wwwroot/controllers/PlanCtrl.js

```diff
@@ -47,6 +47,11 @@
         vm.allowChangePlan = !response.data.endDate;
         vm.hideDragMe = !vm.isFreeTrial;
         defaultPlanDeliveries = !vm.isFreeTrial ? response.data.includedDeliveries.toString() : defaultPlanDeliveries; 
+        if (!vm.allowChangePlan) {
+          settings.getNextPlan().then(function(response){
+            vm.nextPlan = response.data.includedDeliveries;
```

Comment:
It is not the name of the plan, what happen if it is PRO or something like that?

Reply:
Yes, we are going to show only the amount of deliveries allowed for the plan the user select, we are currently not showing the name of the current plan.

Reply:
In that case maybe _nextPlan_ is not the best name